{
  "apiGeneralError": {
    "user-already-exists": "User already exists",
    "Internal server error": "Internal server error",
    "wrong-credentials": "Wrong credentials",
    "account-not-confirmed": "Account has not been confirmed",
    "unauthorized": "Unauthorized",
    "session-has-expired": "Session has expired"
  },
  "generalError": {},
  "unknownError": {
    "unknownError": "Unknown error"
  }
}
